contents/blog/the-posthog-array-1-30-0.md

```diff
@@ -0,0 +1,130 @@
+---
+date: 2021-11-17
+title: Array 1.30.0
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ["Release notes"]
+featuredImage: ../images/blog/array/1-28-0.png
+excerpt: Excerpt here
+---
+
+
+PostHog 1.30.0 includes a description here.
+
+## PostHog 1.30.0 release notes
+
+> For a better experience and new features, remember to [update your PostHog instance](/docs/self-host/configure/upgrading-posthog).
+
+**Release highlights:**
+- [Fresh new look-and-feel](#fresh-new-look-and-feel)
+- [Correlation analysis](#correlation-analysis)
+- [Saved insights](#saved-insights)
+- [Fully revamped recordings](#fully-revamped-recordings)
+
+### Fresh new look-and-feel
+
+PostHog just got a lot fresher! We have a brand new UI and layout that had been on the works, codename `lemonade` (because it's fresh). It's not only a coat of paint, we've pushed a lot of improvements to overall navigation and performance, we call it turbo mode.
+
+
+{images here}
+
+<br />
+
+### Correlation analysis
+
+Trying to understand why your users are converting or dropping? Presenting Correlation analysis. This nifty new insight will automatically match your funnels to any relevant signals on why your users convert or drop (e.g. "Users in Canada are 5x more likely to convert" or "Users in Chrome are 3x less likely to convert"). Funnel optimization to the next level. [Read more here](/docs/....).
+
+{image here}
+
+> 🎁 Correlation analysis is a premium feature and requires a PostHog Scale or Enterprise license. [Learn more](/pricing).
+
+<br />
+
+### Saved insights
+
+No more recreating the same insight multiple times. You will now be able to save insights on PostHog without adding them to a dashboard. Further, you'll be able to see a list of insights created by other team members, and easily search them and filter them. We're investing a lot of effort to make sure you can better collaborate within PostHog.
+
+{image here}
+
+<br />
+
+### Fully revamped recordings
+
+The recordings playback experience just got a lot better. From significant performance improvements (you'll no longer need to wait for the entire recording to load), to a brand new playback interface. Find the right spot in a recording quickly and understand better what your users are doing.
+
+<img src="https://posthog-static-files.s3.us-east-2.amazonaws.com/Website-Assets/Array/1_30_0-recordings.png" alt="Screenshot of new recordings playback experience" />
+
+<br />
+
```

Comment:
@macobo do you want to include something about groups? maybe to allow users to start capturing group info?

Reply:
Decision is up to you both, but @macobo and I are chatting later this week about a launch for Groups. My expectation at the moment is that I'll work on a separate blog post for it, probably writing next week.

Reply:
We'll handle it later. This release we'll be testing with alpha users.